ISM43362/ISM43362.cpp

```diff
@@ -196,6 +196,61 @@ bool ISM43362::dhcp(bool enabled)
     return (_parser.send("C4=%d", enabled ? 1 : 0) && check_response());
 }
 
+/*
+  * @brief  Update modem_country_code.
```

Comment:
everywhere else in the file, the comments refer to WIFI module not modem.
Let's be consistent, either changer everything to modem, or use module

Reply:
Yes I am choosing WIFI module.